### ISSUE TITLE
\Phalcon\Session\Adapter\Database.php Garbage collector

### ISSUE THREAD

**Issue by michaelotte** [phalcon/incubator#248]
The Garbage Collector gc-function should also delete created but not modified sessions:

```
sprintf(
    'DELETE FROM %s WHERE COALESCE(%s, %s) + %d < ?',
    $options['db']->escapeIdentifier($options['table']),
    $options['db']->escapeIdentifier($options['column_modified_at']),
    $options['db']->escapeIdentifier($options['column_created_at']),
    $maxlifetime
)
```

**Comment by noobiwan** [comment]
@michaelotte, @sergeyklay, why is this a bug? I'm working with the adapter and I might be able to fix this if you give me more details.

**Comment by michaelotte** [comment]
Thanks, but this was actually fixed in ebec4a5. Can be closed.(https://github.com/phalcon/incubator/commit/ebec4a5dcbb75930da8ab8f6612cd614a715fc4f)
Before the code did not respect sessions that were just created and not yet modified.
